src/DocumentFormat.OpenXml/GeneratedCode/schemas_microsoft_com_office_drawing_2014_chartex.g.cs

```diff
@@ -12,6 +13,270 @@
+        {
+            get => GetElement<ExtensionList>();
+            set => SetElement(value);
+        }
+
+        private static readonly CompiledParticle _constraint = new CompositeParticle(ParticleType.Sequence, 1, 1)
+        {
+            new ElementParticle(typeof(DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing.ChartData), 1, 1, version: FileFormatVersions.Office2016),
+            new ElementParticle(typeof(DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing.Chart), 1, 1, version: FileFormatVersions.Office2016),
+            new ElementParticle(typeof(DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing.ShapeProperties), 0, 1, version: FileFormatVersions.Office2016),
+            new ElementParticle(typeof(DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing.TxPrTextBody), 0, 1, version: FileFormatVersions.Office2016),
+            new ElementParticle(typeof(DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing.ColorMappingType), 0, 1, version: FileFormatVersions.Office2016),
+            new ElementParticle(typeof(DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing.PrintSettings), 0, 1, version: FileFormatVersions.Office2016),
+            new ElementParticle(typeof(DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing.ExtensionList), 0, 1, version: FileFormatVersions.Office2016)
+        }.Compile();
+
+        internal override CompiledParticle CompiledParticle => _constraint;
+
+        /// <inheritdoc/>
+        public override OpenXmlElement CloneNode(bool deep) => CloneImp<ChartSpace>(deep);
+
+        internal ChartSpace(ExtendedChartPart ownerPart) : base(ownerPart)
+        {
+        }
+
+        /// <summary>
+        /// Loads the DOM from the ExtendedChartPart
+        /// </summary>
+        /// <param name="openXmlPart">Specifies the part to be loaded.</param>
+        public void Load(ExtendedChartPart openXmlPart)
+        {
+            LoadFromPart(openXmlPart);
+        }
+
+        /// <summary>
+        /// Saves the DOM into the ExtendedChartPart.
+        /// </summary>
+        /// <param name="openXmlPart">Specifies the part to save to.</param>
+        public void Save(ExtendedChartPart openXmlPart)
+        {
+            base.SaveToPart(openXmlPart);
+        }
+
+        /// <summary>
+        /// Gets the ExtendedChartPart associated with this element.
+        /// </summary>
+        public ExtendedChartPart ExtendedChartPart
+        {
+            get => OpenXmlPart as ExtendedChartPart;
+            internal set => OpenXmlPart = value;
+        }
+    }
+
+    /// <summary>
+    /// <para>Defines the RelId Class.</para>
+    /// <para>This class is available in Office 2016 or above.</para>
+    /// <para>When the object is serialized out as xml, it's qualified name is cx:chart.</para>
+    /// </summary>
+    [SchemaAttr(80, "chart")]
+    [OfficeAvailability(FileFormatVersions.Office2016)]
+    public partial class RelId : OpenXmlLeafElement
```

Comment:
Is this used? I don't think it needs to be generated.

Reply:
removed